docs/getting-started.md

```diff
@@ -6,14 +6,14 @@ title: Getting Started
 
 Om aan te sluiten kun je de volgende stappen doorlopen:
 1. [Meld je aan bij het kadaster om toegang te krijgen tot de productieomgeving](#aanmelden-om-aan-te-sluiten)
-2. [Bekijk de functionaliteit en specificaties](#functionaliteit)
-3. [Probeer en test de API in de productie-omgeving](#probeer-en-test-de-api-in-de-productie-omgeving)
-
+2. [Bekijk de functionaliteit en specificaties](#functionaliteit-en-specificaties)
+3. [Implementeer de API](#bouw-de-api)
+4. [Probeer en test de API](#probeer-en-test-de-api)
 
 ## Aanmelden om aan te sluiten
 [Vraag een API key voor de productieomgeving aan](https://formulieren.kadaster.nl/aanvraag_bag_api_huidige_bevragingen_productie){:target="_blank"}. Testen doe je bij voorkeur in de productie-omgeving.
```

Comment:
Graag aanpassen in:' Testen doe je bij voorkeur in de testomgeving.'
Door testen in de productieomgeving worden ongecontroleerde aantallen requests ontvangen door nog niet gevonden fouten in de code die bijvoorbeeld loops veroorzaken. Hierdoor kan de beschikbaarheid van de API voor alle klanten onder druk komen te staan. 
En idem als bij regel 8.

Reply:
@NicoleKortoomsBAG ik herinner me dat we hebben afgesproken dat gebruikers in productie kunnen testen. 

De acceptatieomgeving die ik gebruik voor nieuwe releases kunnen aansluiters niet gebruiken omdat die niet stabiel hoeft te zijn. Daarop komen niet volledig geteste nieuwe versies. Is er ook een testomgeving (eto) die altijd gelijk is aan productie waar aansluiters op kunnen testen? De aansluiting daarop (ten minste de url) moet dan ook worden toegevoegd aan de Getting started.

Reply:
> ongecontroleerde aantallen requests ontvangen door nog niet gevonden fouten in de code die bijvoorbeeld loops veroorzaken

Dat kan m.i. ook worden opgelost met rate limiting. Dat zouden we kunnen toevoegen als dat gewenst is. Zie https://geonovum.github.io/KP-APIs/API-strategie-extensies/#rate-limiting

Reply:
Er mag getest worden in productie, we kunnen ook niet zien of iets een testgeval is of een productievraag, maar de voorkeur is om in de acceptatieomgeving te testen. Als straks rate en quota limiting wordt toegepast, is het voor de afnemer nog belangrijker dat hij test in de testomgeving. Anders verbruikt hij zijn quota door testen. 
Als de afnemer iets specifieks wil testen en de testomgeving dat geval niet kent, zal dat enkele geval geen probleem zijn voor zijn quota. Als hij nieuwe code test en er zit een loop in, kan hij zijn eigen productie stilleggen. Door de client het direct goed in te laten richten, voorkomen dat en dat hij later extra werk moet doen.

Reply:
bij BRK hebben we naast productie acceptatieomgeving waar wij (Haal Centraal) nieuwe versies van de API op testen, en de ETO waar aansluiters op testen. De ETO is dus technisch altijd gelijk aan productie, maar bevat de data van de acceptatieomgeving. Dat is wat je voor BAG ook ook nodig hebt, wanneer je niet wilt dat op productie wordt getest.

Reply:
Ik heb hiervoor #533 aangemaakt